lib/APIPlugin.js

```diff
@@ -37,17 +37,11 @@ class APIPlugin {
 
 			params.normalModuleFactory.plugin("parser", parser => {
 				Object.keys(REPLACEMENTS).forEach(key => {
-					parser.plugin(`expression ${key}`, expr => {
-						const dep = new ConstDependency(REPLACEMENTS[key], expr.range);
-						dep.loc = expr.loc;
-						parser.state.current.addDependency(dep);
-						return true;
-					});
-
+					parser.plugin(`expression ${key}`, ParserHelpers.toConstantDependency(REPLACEMENTS[key]));
 					parser.plugin(`evaluate typeof ${key}`, ParserHelpers.evaluateToString(REPLACEMENT_TYPES[key]));
 				});
 				IGNORES.forEach(key => {
-					parser.plugin(key, () => true);
+					parser.plugin(key, ParserHelpers.returnTrue);
```

Comment:
well I guess the only problem with `skipTraversal` is that it is only true in 3 cases, and in the other cases it just "confirmes" `can-rename`.
Would it be ok for me to add two helper functions, one `approve` and one `skip` so they semantically work? given that a simple `() => true` can have these meanings make the code hard to read imho